qutebrowser/browser/commands.py

```diff
@@ -1667,7 +1666,9 @@ def on_file_updated(self, elem, text):
             elem.set_value(text)
         except webelem.OrphanedError as e:
             message.error('Edited element vanished')
+            ed.backup()
         except webelem.Error as e:
+            ed.backup()
             raise cmdexc.CommandError(str(e))
```

Comment:
Probably would make sense to make those two consistent - i.e. do `ed.backup()` and then `raise cmdexc.CommandError` at both places, or use `message.error` and then `ed.backup()` at both places.

Reply:
I'll switch both to `message.error`, as I think the 'backup at ...' message should come after the error